lua/lsp/tsserver-ls.lua

```diff
@@ -1,15 +1,28 @@
 vim.cmd "let proj = FindRootDirectory()"
 local root_dir = vim.api.nvim_get_var "proj"
 
--- use the global prettier if you didn't find the local one
-local prettier_instance = root_dir .. "/node_modules/.bin/prettier"
-if vim.fn.executable(prettier_instance) ~= 1 then
-  prettier_instance = O.lang.tsserver.formatter.exe
+local formatter_exe = O.lang.tsserver.formatter.exe
+
+local get_prettier_module_name = function()
+  -- get local module name
+  if formatter_exe == "prettierd" then
+    return "@fsouza/prettierd"
+  end
+  return formatter_exe
+end
```

Comment:
You don't need this ! The local module name is still
`node_modules/.bin/prettierd` and not `node_modules/.bin/@fsouza/prettierd` !

Reply:
Alright thank you for this! 😄